back-office/multistoregeneralspecs.md

```diff
@@ -0,0 +1,192 @@
+# SPECS GENERAL MULTISTORE BACK-OFFICE
+
+As a merchant I need to be able to correctly manage all my stores
+
+## WHAT
+
+Thus, this project aims at specifying the current and expected behaviour when the multistore is enabled and when there are many stores. It's the general behavior. Some pages can have specific behavior, you can find these specials cases in back-office/multistorespecialsspecs.md
+
+## SPECIFICATIONS
+
+### Shop & shop group color
+
+While creating a shop or a group, the user can select a color as described in the specs to setup the multistore<br/>
+This color will be displayed in the multistore header for the selected shop or shop group.<br/>
+It will be also displayed before the name of each shop / group in the context drop-down.
+
+### Multistore header
+
+On each pages, when the multistore feature is enabled and if many shops are created, a multistore header is displayed.<br/>
+The multistore header contains many things listed below:
+
+**Color**
+
+This header takes the color of the selected context:<br/>
+- The all shops context is blue and this color can't be changed by the user.<br/>
+- The shop group color can be configured while creating or editing the group<br/>
```

Comment:
remind that the color must be selected in the Multistore configuration page (+ Link to the specs)